static/js/containers/CollectionListPage.js

```diff
@@ -59,6 +59,9 @@ class CollectionListPage extends React.Component {
             <Link to={makeCollectionUrl(collection.key)}>
               {collection.title}
             </Link>
+            <span className="mdc-list-item__text__secondary">
+              {collection.video_count} Videos
+            </span>
```

Comment:
it looks like there aren't any tests for this whole file - should we add one now? it would be good to have I think, although if we don't want to take of it in this PR we should just make sure that this is tested when we do add it

Reply:
I added tests for this file in my PR #240. Maybe you could copy that over, remove the test specific to my PR and add one specific to this one